Task08/AveragePrice.java

```diff
@@ -0,0 +1,26 @@
+package test.HW8;
+
+/**
+ * Class contains method for calculating average price of products.
+ */
+public class AveragePrice extends PrimaryCommandClass {
+    /**
+     * Method calculates average price of products
+     *
+     * @param command string of command.
+     * @param cd      data stores object.
+     * @see PrimaryCommandClass#commandName(String, CreateData)
+     */
+    @Override
+    public void commandName(String command, CreateData cd) {
```

Comment:
It's very strange to see that method named commandName (I suppose it must return command name) actually preform check and command execution - please try to find more obvious name for the things